.circleci/config.yml

```diff
@@ -1,69 +1,62 @@
-version: 2
-jobs:
-  test-python3:
+version: 2.1
+
+executors:
+  tox:
     docker:
-      - image: circleci/python:3.6.7
```

Comment:
Do you think we could run into any issues with CircleCI in not using their pre-baked images? (Not that I can think of any per se, but worth making sure)

Reply:
@brianmartin regarding above comment: any thoughts on using the circleci python container instead and adding a `pip install tox` step first?

Reply:
We certainly could use the circle image. The nice thing about this tox container is that it contains the full set of python versions. Any reason to prefer theirs? I'm curious what your concern is, as using this tox image seems simpler. Is it a trust concern?